pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -400,6 +400,14 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 		cr.Spec.Backup.PITR.Enabled = false
 	}
 
+	if cr.Spec.Backup.PITR.Enabled {
+		if len(cr.Spec.Backup.Storages) != 1 {
```

Comment:
It will not work with the second case.
`If the user had one bucket and pitr enabled and now adds one more bucket, PITR continues to work, we throw an error in the operator logs:`
Yea, PITR will continue to work, but operator will not, because we always return there and error in that case, and operator will not process any CR changes

Reply:
Fixed in 54a19611247bded4fe8832a9f54558be0a45f1ab